translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/ProjectManager/activities/ActivityUnitList.java

```diff
@@ -104,7 +104,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
 
             // Set its layout manager
             mLayoutManager = new LinearLayoutManager(this);
-            mLayoutManager.setOrientation(LinearLayoutManager.VERTICAL);
+            mLayoutManager.setOrientation(RecyclerView.VERTICAL);
```

Comment:
same as previous PR

Reply:
@sarabiaj Reverted to using LinearLayoutManager